_posts/2020-11-08-convert-wide-dataset-to-long.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title:  "Converting Wide Dataset to Long Format"
+date:   2020-11-08 04:39:04 -0700
+categories: wide dataset long pivot_longer tidy
+---
+# Converting Wide Dataset to Long Format
+
+Completed by Kathy Anh Lam              
+November 8, 2020                      
+kplam4@usfca.edu
```

Comment:
I'm not sure if you want to include your email on the public class website. Just a note, but completely up to you!

Reply:
Fixed with b097bbadcca2c88aebd9f9648fbdd612a48d259d